### PR TITLE
Update link to documentation in PR template

### PR DESCRIPTION
**Pull request by stoneman**
Edit: Description below has been updated. The PR originally updated links in the language files too.

This link is 404ing as there is no `/docs` directory in the `betaflight` repo. I'm guessing those docs were moved to `betaflight.com` so I've updated the link to what I assume is the correct location within https://betaflight.com.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by HThuren** [comment]
I belive the language file update must be in Crowdin

**Comment by haslinghuis** [comment]
We only need to update `en` locale. Translations are done using crowdin

**Comment by stoneman** [comment]
Changes to language files reverted.

For reference, it's the it, pl & pt_BR files that contain out of date links.

**Comment by HThuren** [comment]
> Changes to language files reverted.
> 
> For reference, it's the it, pl & pt_BR files that contain out of date links.

Maybe @McGiverGim can notify the translators (I don't know howto) ?

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
